Hook-into-PlaceholderAPI.md

```diff
@@ -94,7 +94,7 @@ Next step is to go to your plugin.yml or paper-plugin.yml and add PlaceholderAPI
   author: author
   main: your.main.path.Here
   
-  # This sets PlaceholderAPI as an optional dependency for your plugin.
+  # This sets PlaceholderAPI as a dependency for your plugin.
```

Comment:
What about adding that it makes it required?
Would that be too much?

Reply:
It's hard to say which is better. In my opinion, "mandatory" sounds better than "required", but perhaps it's still too much.

Reply:
Yeah. The current change is fine enough. We can consider a change for the future perhaps...